video_xblock/static/js/videojs-transcript.js

```diff
@@ -1,9 +1,28 @@
 domReady(function() {
-  videojs('{{ video_player_id }}').ready(function(){
+  videojs('{{ video_player_id }}').ready(function() {
+
+    var tracks = this.textTracks();
+    var enableTrack = false;
+    for (var i = 0; i < tracks.length; i++) {
```

Comment:
You do not need index here in the loop. Please use foreach, and also consider use  underscore.js utility functions.